_source/logzio_collections/_prometheus-sources/dotnet-custom.md

```diff
@@ -0,0 +1,555 @@
+
+            Thread.Sleep(10000);
+
+            metrics.Measure.Counter.Increment(counter);
+
+            Thread.Sleep(100000); 
+```
+
+In the above example, the metric has a name ("my_counter"), a tag key ("test") and a tag value ("my_test"). These parameters are used to query data from this metric in your Logz.io dashboard.
+
+
+###### Apdex
+
+Apdex (Application Performance Index) allows us to monitor end-user satisfaction. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/apdex/).
+
+###### Counter
+
+Counters are one of the simpliest metrics types supported and allow us to track how many times something has happened. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/counters/).
+
+###### Gauge
+
+A Gauge is simply an action that returns the instantaneous measure of a value, where the value abitrarily increases and decreases, for example CPU usage. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/gauges/).
+
+###### Histogram
+
+Histograms measure the statistical distribution of a set of values. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/histograms/).
+
+###### Meter
+
+A Meter measures the rate at which an event occurs along with a total count of the occurances. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/meters/).
+
+###### Timer
+
+A Timer is a combination of a Histogram and a Meter allowing us to measure the duration of a type of event, the rate of its occurrence and provide duration statistics. For more information on this metric, refer to [App Metrics documentation](https://www.app-metrics.io/getting-started/metric-types/timers/).
+
+
+##### Run your application
+
+Run your application to start sending metrics to Logz.io.
+
+
+##### Check Logz.io for your events
+
+Give your events some time to get from your system to ours, and then open [Metrics dashboard](https://app.logz.io/#/dashboard/metrics/discover?).
+
+##### Filter the metrics by labels
+
+Once the metrics are in Logz.io, you can query requited metrics using labels. Each metric has the following labels:
+
+| App Metrics parameter name | Description | Logz.io parameter name |
+|---|---|---|
+| Name | The name of the metric. Required for each metric. | Metric name if not stated otherwise |
+| MeasurementUnit | The unit you use to measure. By default it is `None`. | `unit` |
+| Context | The context which the metric belong to. By default it is `Application`.	 | `context` |
+| Tags | Pairs of key and value of the metric. It is not required to have tags for a metric.| Tags keys |
+
+Some of the metrics have custom labels as described below.
```

Comment:
You can write here that the clients should replace [[your_metric_name]] with their metric name.